### PR TITLE
fix: handle reopening quiz after partial attempt

### PR DESCRIPTION
**Pull request by KaustubhKumar05**

### CODE REVIEW

File: packages/roomkit-react/src/Prebuilt/components/Polls/Voting/PeerParticipationSummary.tsx [hunk 1]
```diff
@@ -33,12 +33,12 @@ export const PeerParticipationSummary = ({ quiz }: { quiz: HMSPoll }) => {
         { title: 'Avg. Time Taken', value: getFormattedTime(summary.avgTime) },
         {
           title: 'Avg. Score',
-          value: Number.isInteger(summary.avgScore) ? summary.avgScore : summary.avgScore.toFixed(2),
+          value: Number.isInteger(summary.avgScore) ? summary.avgScore : summary.avgScore.toFixed(2) || 0,
```
Comment (raviteja83) [inline packages/roomkit-react/src/Prebuilt/components/Polls/Voting/PeerParticipationSummary.tsx:36]: why is there no `?.` if we are doing || ?

Comment (KaustubhKumar05) [reply packages/roomkit-react/src/Prebuilt/components/Polls/Voting/PeerParticipationSummary.tsx:36]: summary is defined, avgscore can be undefined

Comment (KaustubhKumar05) [reply packages/roomkit-react/src/Prebuilt/components/Polls/Voting/PeerParticipationSummary.tsx:36]: This is just needed for the score, will remove for avgScore and check

File: packages/roomkit-react/src/Prebuilt/components/Polls/common/SingleChoiceOptions.jsx [hunk 2]
```diff
@@ -17,10 +17,9 @@ export const SingleChoiceOptions = ({
   isStopped,
   isQuiz,
   localPeerResponse,
-  answer,
 }) => {
   return (
-    <RadioGroup.Root value={answer || null} onValueChange={value => setAnswer(value)}>
+    <RadioGroup.Root value={localPeerResponse?.option} onValueChange={value => setAnswer(value)}>
```
Comment (raviteja83) [inline packages/roomkit-react/src/Prebuilt/components/Polls/common/SingleChoiceOptions.jsx:22]: shouldn't this be || null?

Comment (KaustubhKumar05) [reply packages/roomkit-react/src/Prebuilt/components/Polls/common/SingleChoiceOptions.jsx:22]: Not requried now since multiple cards need to rendered

**Review by raviteja83 — APPROVED** [review]

**Pull request merged**